### PR TITLE
Included pin and address commands.

### PR DESCRIPTION
**Pull request by ronaldsteen**
Hi,

I've included a couple new commands to specify pin and addresses.

DST [pin] sensors
Get all connected sensors' addresses

DST [pin] [address]
Get a single temperature C

DST [pin] all
Get all sensors' temperatures on that pin.

Cheers,
Ronald

### CODE REVIEW

File: homeduino.ino [hunk 1]
```diff
@@ -1,8 +1,9 @@
 #include <SerialCommand.h>
 #include <OneWire.h>
 #include <DallasTemperature.h>
+#include <stdlib.h>
 
-#define ONE_WIRE_BUS 2
+#define ONE_WIRE_BUS 12
```
Comment (sweetpi) [inline homeduino.ino]: This defined isn't used anymore, right?

Comment (ronaldsteen) [inline homeduino.ino]: That's correct

**Comment by sweetpi** [comment]
Looks much much better. Very nice!

**Comment by ronaldsteen** [comment]
Thanks. You're welcome.

I'm also updating the other packages to profit from these changes.

It's currently running fine on my pi for a night.

**Comment by sweetpi** [comment]
Nice, let me know when you want to have it merged.

**Pull request merged**